rtl/eclrtl/rtldynfield.cpp

```diff
@@ -1099,9 +1109,9 @@ class GeneralRecordTranslator : public CInterfaceOf<IDynamicTransform>
         unsigned numOffsets = sourceRecInfo.getNumVarFields() + 1;
         size_t * variableOffsets = (size_t *)alloca(numOffsets * sizeof(size_t));
         RtlRow sourceRow(sourceRecInfo, sourceRec, numOffsets, variableOffsets);  // MORE - could save the max source offset we actually need, and only set up that many...
-        return doTranslate(builder, callback, offset, sourceRow);
+        return doTranslateOpaqueType(builder, callback, offset, &sourceRow);
     }
-    size32_t doTranslate(ARowBuilder &builder, IVirtualFieldCallback & callback, size32_t offset, const RtlRow &sourceRow) const
+    size32_t doTranslateOpaqueType(ARowBuilder &builder, IVirtualFieldCallback & callback, size32_t offset, const void *sourceRow) const
```

Comment:
can be renamed back to doTranslate

Reply:
would clash with line 1107, that deals with binary rows only.

This method deals with a 'sourceRow' that can either be a binary source row, or a dynamic callback.

Reply:
I think I'll leave as is.